src/VisualStudio/TestSetup/IntegrationTestServiceCommands.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Design;
+using System.Diagnostics;
+using System.Runtime.Remoting;
+using System.Runtime.Remoting.Channels;
+using System.Runtime.Remoting.Channels.Ipc;
+using System.Runtime.Serialization.Formatters;
+using Microsoft.VisualStudio.Shell;
+using Roslyn.VisualStudio.Test.Utilities;
+
+namespace Roslyn.VisualStudio.Test.Setup
+{
+    internal sealed class IntegrationTestServiceCommands : IDisposable
+    {
+        public const int cmdidStartIntegrationTestService = 0x0100;
+        public const int cmdidStopIntegrationTestService = 0x0101;
+
+        public static readonly Guid grpidIntegrationTestServiceCommands = new Guid("82A24540-AEBC-4883-A717-5317F0C0DAE9");
```

Comment:
All `public` members should be pascal cased vs. camel cased (all 3 definitions here).